mesheryctl/internal/cli/root/config/config.go

```diff
@@ -81,16 +81,24 @@ func (ctx *Context) ValidateVersion() error {
 		return nil
 	}
 
-	matched, err := regexp.MatchString(`v[0-9]+.[0-9]+.[0-9]+[-a-z0-9]*`, ctx.Version)
-	if err != nil || !matched {
-		return errors.New("invalid version " + ctx.Version + " specified")
+	url := "https://api.github.com/repos/layer5io/meshery/git/trees/" + ctx.Version + "?recursive=1"
```

Comment:
We'll want to refer to these constants - https://github.com/layer5io/meshery/blob/57f4e901d19acdc9c2586e04b57001ace67929f2/mesheryctl/pkg/utils/helpers.go#L60-L62

Speaking of these constants, having them in a utils file called "helpers" doesn't seem the most befitting.

Reply:
Agreed @leecalcote 
But I'm not sure how to use these constants in `config.go` since that's imported in `pkg/utils` and hence using these(via the getter functions) leads to circular imports.    

I have used these constants in a URL in another function in `platform.go` though.